zulipterminal/ui_tools/buttons.py

```diff
@@ -469,6 +469,8 @@ def handle_link(self, *_: Any) -> None:
             if self.controller.is_any_popup_open():
                 self.controller.exit_popup()
             process_media(self.controller, self.link)
+        elif self.link.startswith(("https://", "http://")):
+            self.controller.open_in_browser(self.link)
```

Comment:
It would be useful to define the protocols we support somewhere as a constant rather than inline here.